spec/shared_context/users.rb

```diff
@@ -74,14 +74,14 @@
     # failed payments on nov 29 THIS_YEAR
     Timecop.freeze(nov_29) do
       create(:membership_fee_payment,
-             :expired,
```

Comment:
This should not be removed.

You are changing the intention of the test; see the comment above the Timecop statuement.

See my comment in `spec/factories/payments.rb`

We need to have tests to ensure that we can handle (old) HIPS payments that still exist in the system. Likewise with the /spec tests: Don't remove any tests that created / used :expired or :awaiting_payment because we need to ensure that we can still deal with HIPS payments in that are in the system.